gcp_variant_transforms/testing/integration/medium_tests/test_1000_copies_of_small_files_with_merge.json

```diff
@@ -0,0 +1,24 @@
+{
+  "test_name": "test-1000-copies-of-small-files-with-merge",
+  "table_name": "test_1000_copies_of_small_files_with_merge",
+  "input_pattern": "gs://gcp-variant-transforms-testfiles/medium_tests/valid-1000-copies/*.vcf",
+  "variant_merge_strategy": "MOVE_TO_CALLS",
+  "runner": "DataflowRunner",
+  "worker_machine_type": "n1-standard-16",
+  "max_num_workers": "20",
+  "num_workers": "20",
+  "assertion_configs": [
```

Comment:
Please also add a check for counting number of calls for a specific position and for all positions, e.g.,
`
SELECT COUNT(0)
FROM integration_tests_20180309_145140.test_1000_copies_of_small_files_with_merge AS T, T.call AS call
WHERE start_position = 0
`
This is one more sanity check for merging.

Reply:
Done.